components/builder-web/app/BuilderApiClient.ts

```diff
@@ -620,12 +620,15 @@ export class BuilderApiClient {
     });
   }
 
-  public validateDockerCredentials(username: string, password: string) {
+  public validateIntegrationCredentials(username: string, password: string, url?: string) {
+    let creds = { username, password };
+    if (url && url !== '') creds['url'] = url;
```

Comment:
I usually `.trim()` here too just in case a space snuck in there somehow.

Reply:
Great pro-tip!